zeppelin-web/src/components/navbar/navbar.html

```diff
@@ -83,19 +83,65 @@
               </span>
             </div>
           </form>
-        <li class="server-status"  >
+        <li class="server-status">
+        <div class="dropdown">
           <i class="fa fa-circle" ng-class="{'server-connected':navbar.connected, 'server-disconnected':!navbar.connected}"></i>
-        <span ng-show="navbar.connected" ng-if="ticket.principal == 'anonymous' ">Connected</span>
-        <span ng-show="navbar.connected" ng-if="ticket.principal != 'anonymous' " tooltip-placement="bottom" tooltip="{{fullUsername}}">{{truncatedUsername}}</span>
-        <span ng-show="!navbar.connected">Disconnected</span>
+          <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown"
+                  ng-show="navbar.connected" ng-if="ticket.principal == 'anonymous' ">Connected
+            <span class="caret"></span></button>
+          <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown"
+                  ng-show="navbar.connected" ng-if="ticket.principal != 'anonymous' "
+                  tooltip-placement="bottom" tooltip="{{fullUsername}}">{{truncatedUsername}}
```

Comment:
@ravicodder Ah ha I see. I didn't thought about that case. Thanks for letting me know :)